<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicSoundTouchWidget</class>
 <widget class="QWidget" name="MusicSoundTouchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>音色变音</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>480</width>
     <height>330</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>4</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>音色变音</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>462</x>
     <y>6</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>29</y>
     <width>480</width>
     <height>305</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>49</y>
     <width>311</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{border:1px solid #555;}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>打开WAV</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>打开WAV文件或者开始录音</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>结束录音</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>开始录音</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>154</y>
     <width>311</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{border:1px solid #555;}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="MusicClickedSlider" name="tempoSlider">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>20</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="tempoLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>速度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="pitchLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>音色：</string>
    </property>
   </widget>
   <widget class="MusicClickedSlider" name="pitchSlider">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>50</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="rateLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>比率：</string>
    </property>
   </widget>
   <widget class="MusicClickedSlider" name="rateSlider">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>80</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="pitchLabelValue">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="rateLabelValue">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="tempoLabelValue">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="pathLabel">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>284</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="transformButton">
   <property name="geometry">
    <rect>
     <x>374</x>
     <y>190</y>
     <width>85</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>开始转换</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playWavButton">
   <property name="geometry">
    <rect>
     <x>374</x>
     <y>240</y>
     <width>85</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>播放转换录音</string>
   </property>
  </widget>
  <widget class="QLabel" name="pathLabels">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>284</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>路径：</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>pathLabel</zorder>
  <zorder>transformButton</zorder>
  <zorder>playWavButton</zorder>
  <zorder>pathLabels</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicClickedSlider</class>
   <extends>QSlider</extends>
   <header>musicclickedslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
